sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,14 +1,25 @@
 # Release History
 
-## 1.6.0-beta.3 (Unreleased)
+## 1.6.0 (2023-10-10)
 
 ### Features Added
 
+- Added `WorkloadIdentityCredential`.
+
 ### Breaking Changes
 
+- Modify the order of the credentials used within the `DefaultAzureCredential` to be consistent with other languages.
```

Comment:
@ahsonkhan, are we ok GA'ing this change with #4952 being active?

Reply:
It would be good to be explicit of the order change and/or to reference the #issue for folks to look up

Reply:
Source code is the documentation.  We explicitly document that we don't guarantee the order or that order stays consistent.  DefaultAzureCredential is a development only scenario in C++.

Reply:
> @ahsonkhan, are we ok GA'ing this change with #4952 being active?

That's a great question Anton!

In general, given this is a dev-time credential for getting started/usability, I am not as worried about it. That said, we should do this, to have consistent semantics across languages, and improve the performance overall:
https://github.com/Azure/azure-sdk-for-cpp/issues/5033